apps/avatax/scripts/migrations/1.1-avatax-migration.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable multiline-comment-style */
+import { checkoutCalculateTaxesSyncWebhook } from "../../src/pages/api/webhooks/checkout-calculate-taxes";
+import { orderCalculateTaxesSyncWebhook } from "../../src/pages/api/webhooks/order-calculate-taxes";
+import { AppWebhookMigrator } from "./app-webhook-migrator";
+
+/**
+ * Contains the migration logic for the Taxes App. In the 1st step, it is expected to only write, not delete. The cleanup will be done in the 2nd step.
+ * @param webhookMigrator - The AppWebhookMigrator instance.
+ */
+export async function migrateAvatax(webhookMigrator: AppWebhookMigrator) {
+  // Migration plan:
+  //  1. Update subscriptionQuery of all calculateTaxes webhooks
+  webhookMigrator.updateWebhookQueryByHandler(orderCalculateTaxesSyncWebhook);
+  webhookMigrator.updateWebhookQueryByHandler(checkoutCalculateTaxesSyncWebhook);
```

Comment:
please verify if it works, probalby easiest to check if staging were updated after we merge this PR

I assume its a quite straight forward logic - to just use latest possible subscription query

Reply:
I just realized it will not work, because we miss "deploy script" in Vercel. 
See Adyen or Search apps